x-pack/legacy/plugins/index_management/public/app/services/documentation.ts

```diff
@@ -51,6 +51,10 @@ class DocumentationService {
     }
     return `${this.esDocsBase}${typeDefinition.documentation[uri]}`;
   };
+
+  public getNullValueLink() {
```

Comment:
Seb expressed some concern previously around defining urls like this, related to the future portability of the mappings editor.

See: https://github.com/elastic/kibana/pull/50500#discussion_r349105460 and https://github.com/elastic/kibana/pull/50500#discussion_r349421030